### ISSUE TITLE
Mouse click not working in some areas/applications

### ISSUE THREAD

**Issue by tomfulghum** [linuxmint/Cinnamon#5939]
I'm having an issue with cinnamon, namely that clicking with my laptop's touchpad is now almost impossible.
In LightDM, everything still works. I can click every button and field without any problems. As soon as I log into cinnamon though, clicks are registered almost nowhere (panel, title bars, etc.). Right clicking on the panel opens the context menu for the desktop, not the panel's.

This only happens in cinnamon 3.2.1. Downgrading to 3.0.7 fixed the issue for me (I didnt't have any intermediate versions cached).

I'm using cinnamon on top of Arch Linux. The laptop is a Dell XPS 13 2016 developer edition.

Edit: After a little more testing, the issue also exists in 3.2.0.

**Comment by tomfulghum** [comment]
After a little more testing, the issue also exists in 3.2.0.

**Comment by JakeStanger** [comment]
Exactly the same issue here on my desktop (also on Arch)

**Comment by Negromancer** [comment]
Same for me, the bar doesn't seem to react to clicking at all. This starts happening only after I open a window (doesn't matter which, tried Firefox, LilyTerm and a bunch of other apps).
Also Arch.

**Comment by leigh123linux** [comment]
Works fine here on fedora, what did the arch packager forget this time.
I suggest you close this report and re-file it at the arch bug tracker instead!

**Comment by lots0logs** [comment]
It's working fine for me on Antergos. Though I'm using our package not Arch's. I'll look into it and see if I can find any differences between our packages (though really there shouldnt be any).

**Comment by leigh123linux** [comment]
There is a ibus issue with muffin but it's been around for a couple of years

https://github.com/linuxmint/muffin/pull/271

**Comment by morph027** [comment]
Same here on Ubuntu Xenial with Cinnamon 3.2.1

**Comment by lestcape** [comment]
Please i know this is probably my stupid assumption, but hit the windows key and try to start the cinnamon-settings from the menu. Then go to the applet category and click the button to reset the default configuration... At less the old WindowsIconList Applet cause this behaviour in cinnamon 3.2, maybe others.

**Comment by chocolateboy** [comment]
> It's working fine for me on Antergos.

The panel becomes unresponsive for me on Antergos (Cinnamon 3.2.1). I haven't noticed any issues with title bars.

**Comment by chocolateboy** [comment]
Older issue: https://github.com/linuxmint/Cinnamon/issues/1934

**Comment by JakeStanger** [comment]
I just installed Cinnamon 3.2.1-4 on a fresh install of Arch and I don't seem to be getting the issue. The issue still exists on my main Arch install.

**Comment by ase1590** [comment]
Wonder if this is a duplicate of my issue: #5984

**Comment by ase1590** [comment]
in #5984, it was found the the cinnamon applet **Window List With App Grouping** was causing the freezing. check to see if you are using that applet.

**Comment by germanfr** [comment]
@ase1590 it happened to me with one applet yesterday and the problem disappeared when I removed it. Now you say I realize. I can't remember which one was because I installed almost everything from the spices LOL.

**Comment by JakeStanger** [comment]
I run Window List With App Grouping so this would make sense.

**Comment by JosephMcc** [comment]
@tomfulghum Can you try resetting your applets to default and seeing if that fixes the issue for you as well?

**Comment by lestcape** [comment]
@JosephMcc, i think no one see my previous comment...

**Comment by JosephMcc** [comment]
@lestcape Some 3rd party applets do seem to be the cause of this issue. I was going to close it but would just like confirmation that's the case for the OP as well.

**Comment by hillexed** [comment]
Just wanted to add that I experienced the exact same issue - right-clicking on the panel would act as though I right-clicked on the desktop - and removing Window List With App Grouping in the Applets menu and restarting cinnamon with Alt-F2 -> R fixed it for me. Thanks!

**Comment by JosephMcc** [comment]
Ok. Closing this since it seems a 3rd party applet was causing this for everyone.

**Comment by tomfulghum** [comment]
Yes, turns out it really was caused by the applet "Window List With App Grouping" in the end. Thanks for the help!
